index.js

```diff
@@ -1,24 +1,45 @@
 const express = require('express')
+const app = express();
 const knex = require('./db/knex.js')
 const cors = require('cors')
-const port = process.env.PORT || 3000
 const bodyParser = require('body-parser')
+const port = process.env.PORT || 3000
+const httpServer = require('http').createServer()
 
+app.set('port', process.env.PORT || 3000)
 
-const app = express();
+const server = app.listen(app.get('port'), () => console.log(`Listening on port ${port}`))
 
-app.set('port', process.env.PORT || 3000)
+const io = require('socket.io')(httpServer, {
+  cors: {
+    origin: '*',
+    allowHeaders: '*',
+    credentials: true
+  }
+})
 
-app.listen(app.get('port'), () => console.log(`Listening on port ${port}`))
+httpServer.listen(3002)
 
+app.use(function(req, res, next) {
+  res.header('Access-Control-Allow-Origin', '*');
+  res.header('Access-Control-Allow-Methods', 'GET,PUT,POST,DELETE');
```

Comment:
Noting to remember researching/testing if these two headers are necessary, as we are only connecting sockets with no database querying.